src/verifier.js

```diff
@@ -54,8 +54,8 @@ function ensureStringClaimMatcher(raw) {
   }
 
   return raw
-    .filter(r => r)
-    .map(r => {
+    .filter((r) => r)
```

Comment:
let's try to avoid so many formatting changes. change the prettier configuration to avoid the arrowParent